Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -742,6 +744,82 @@ public function getPrevMeaningfulToken($index)
     }
 
     /**
+     * Find a sequence of meaningful tokens and returns the array of their locations.
+     *
+     * @param array      $sequence      an array of tokens (same format used by getNextTokenOfKind)
+     * @param int        $start         start index, defaulting to the start of the file.
+     * @param int        $end           end index, defaulting to the end of the file.
```

Comment:
If one want to pass end index - he should give valid index.
If not - set it empty, so null, not -1, which is invalid index.